packages/ui/src/components/Visualization/ContextToolbar/Flows/FlowsList.tsx

```diff
@@ -89,78 +113,80 @@ export const FlowsList: FunctionComponent<IFlowsList> = (props) => {
+                        </Icon>
+                      ) : (
+                        <Icon isInline>
+                          <EyeSlashIcon title={`Show ${flow.id}`} data-testid={`toggle-btn-${flow.id}-hidden`} />
+                        </Icon>
+                      )
+                    }
+                    variant="plain"
+                    onClick={(event) => {
+                      visualFlowsApi.toggleFlowVisible(flow.id);
+                      /** Required to avoid closing the Dropdown after clicking in the icon */
+                      event.stopPropagation();
+                    }}
+                  />
+                </Td>
 
-              <Td dataLabel={columnNames.current.delete}>
-                <Button
-                  title={`Delete ${flow.id}`}
-                  data-testid={`delete-btn-${flow.id}`}
-                  icon={<TrashIcon />}
-                  variant="plain"
-                  onClick={async (event) => {
-                    const isDeleteConfirmed = await deleteModalContext?.actionConfirmation({
-                      title:
-                        "Do you want to delete the '" +
-                        flow.toVizNode().getId() +
-                        "' " +
-                        flow.toVizNode().getNodeTitle() +
-                        '?',
-                      text: 'All steps will be lost.',
-                    });
+                <Td dataLabel={columnNames.current.delete}>
+                  <Button
+                    title={`Delete ${flow.id}`}
+                    data-testid={`delete-btn-${flow.id}`}
+                    icon={<TrashIcon />}
+                    variant="plain"
+                    onClick={async (event) => {
+                      const isDeleteConfirmed = await deleteModalContext?.actionConfirmation({
+                        title:
+                          "Do you want to delete the '" +
+                          flow.toVizNode().getId() +
+                          "' " +
+                          flow.toVizNode().getNodeTitle() +
+                          '?',
+                        text: 'All steps will be lost.',
+                      });
 
-                    if (isDeleteConfirmed !== ACTION_ID_CONFIRM) return;
+                      if (isDeleteConfirmed !== ACTION_ID_CONFIRM) return;
 
-                    camelResource.removeEntity(flow.id);
-                    updateEntitiesFromCamelResource();
-                    /** Required to avoid closing the Dropdown after clicking in the icon */
-                    event.stopPropagation();
-                  }}
-                />
-              </Td>
-            </Tr>
-          ))}
+                      camelResource.removeEntity(flow.id);
+                      updateEntitiesFromCamelResource();
+                      /** Required to avoid closing the Dropdown after clicking in the icon */
+                      event.stopPropagation();
```

Comment:
I saw this line being removed in a previous PR, could we please rebase this PR so we don't reintroduce the change? :smiley: 

```suggestion
```